.ci/aarch64/lib_install_qemu_aarch64.sh

```diff
@@ -51,14 +50,13 @@ build_and_install_qemu() {
         PACKAGING_REPO="github.com/kata-containers/packaging"
         QEMU_CONFIG_SCRIPT="${GOPATH}/src/${PACKAGING_REPO}/scripts/configure-hypervisor.sh"
 
-        git clone --branch "$CURRENT_QEMU_BRANCH" --single-branch "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
+        git clone --branch "$CURRENT_QEMU_BRANCH" --single-branch --depth 1 --shallow-submodules "${QEMU_REPO_URL}" "${GOPATH}/src/${QEMU_REPO}"
```

Comment:
I was suggesting @nitkon used this on https://github.com/kata-containers/tests/pull/1545#issuecomment-490443025, but that wasn't applied.

I do think this is similar to https://github.com/kata-containers/runtime/pull/1660 - this code is not architecture-specific so we really need a `clone_qemu_repo()` function that accepts as `$branch` param in `.ci/install_qemu.sh` to avoid having to fix the same bug for every architecture.

Reply:
Hi~ @grahamwhaley @jodh-intel I'm fine with spreading it to more archs and use `clone_qemu_repo()` as common function. 
And I'm thinking that if we are using `--depth 1`, we are definitely to be sure that we are building from the HEAD, `git checkout "$CURRENT_QEMU_COMMIT"` is unnecessary and if we failed to update to keep it the same as HEAD, it could cause problem.